packages/@storefront/core/test/unit/services/url.ts

```diff
@@ -347,6 +364,29 @@ suite('URL Service', ({ expect, spy, stub, itShouldBeCore, itShouldExtendBaseSer
       expect(handleUrl).to.be.calledOnce;
     });
 
+    it('should accept a url override', () => {
+      const currUrl  = '/foo';
+      const route = 'search';
+      const state = { a: 'b' };
+      const urlOverride = '/bar';
+      service['app'] = <any>{ flux: { store: { getState: () => state } } };
+      service['emitUrlUpdated'] = spy();
+      service['filterState'] = spy(() => state);
+      service['handleUrl'] = spy();
+      service['opts'] = { redirects: {} };
+      win.history = { pushState: spy(() => win.location = { href: urlOverride } ) };
```

Comment:
Pull `pushState` into a variable and use that variable in the assertion.

Reply:
Noted, will update.